src/domain/usecases/RunPractitionersValidationUseCase.ts

```diff
@@ -264,6 +292,72 @@ export class RunPractitionersValidationUseCase {
         return sumChildren === convertToNumberOrZero(value);
     }
 
+    private isChild1XORChild2(dataElement: DataElementsLevelWithValues): Maybe<
+        DataElementsLevelWithValues & {
+            result: string;
+            child: DataElementWithValue;
+            childPrefix: string;
```

Comment:
Not sure if I'm missing something, but we're not using the `childPrefix` attribute outside this function, right?